docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -112,12 +112,14 @@ configz endpoint:
 
 ```
 $ export NODE_NAME=the-name-of-the-node-you-are-reconfiguring
-$ curl -sSL http://localhost:8001/api/v1/proxy/nodes/${NODE_NAME}/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubeletconfig/v1alpha1"' > kubelet_configz_${NODE_NAME}
+$ curl -sSL http://localhost:8001/api/v1/proxy/nodes/${NODE_NAME}/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubeletconfig/${VERSION}"' > kubelet_configz_${NODE_NAME}
 ```
 
 Note that we have to manually add the `kind` and `apiVersion` to the downloaded
 object, as these are not reported by the configz endpoint. This is one of the
 limitations of the endpoint that is planned to be fixed in the future.
+If you are using a Kubelet less than v1.10, you must substitute `v1alpha1` for `{VERSION}`.
+If you are using a Kubelet v1.10 or higher, you must substitute `v1beta1` for `{VERSION}`.
```

Comment:
You may want to remove the curly brackets above.

Reply:
Do we have a standard format for "values the user should mentally substitute."